include/libspirv/libspirv.h

```diff
@@ -260,6 +260,7 @@ typedef enum spv_validate_options_t {
   SPV_VALIDATE_ID_BIT = SPV_BIT(2),
   SPV_VALIDATE_RULES_BIT = SPV_BIT(3),
   SPV_VALIDATE_SSA_BIT = SPV_BIT(4),
+  SPV_VALIDATE_INSTRUCTION_BIT = SPV_BIT(5) | SPV_VALIDATE_LAYOUT_BIT,
```

Comment:
Before getting to read the code, I assumed at this point that the layout checks passing allow the instruction tests to make simplifying assumptions.   However, if that's the case, then a comment to that effect should be added here.
(And now off to read the further down.)